tests/ocpp_tests/test_sets/ocpp16/ocpp_compliance_tests.py

```diff
@@ -4693,15 +4770,17 @@ async def test_reservation_faulted_state(
 ):
     logging.info("######### test_reservation_faulted_state #########")
 
-    test_controller.diode_fail()
+    test_controller.raise_error("MREC6UnderVoltage", 1)
+
+    await asyncio.sleep(1)
```

Comment:
I dont think that we have to sleep here since `wait_for_and_validate` will wait for the error event already